src/System.EnterpriseServices/src/ApiCompatBaseline.txt

```diff
@@ -1,107 +1 @@
-TypesMustExist : Type 'System.EnterpriseServices.Internal.ComSoapPublishError' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.EnterpriseServices.Internal.GenerateMetadata' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.EnterpriseServices.Internal.IClrObjectFactory' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.EnterpriseServices.Internal.IComManagedImportUtil' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.EnterpriseServices.Internal.IComSoapIISVRoot' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.EnterpriseServices.Internal.IComSoapMetadata' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.EnterpriseServices.Internal.IComSoapPublisher' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.EnterpriseServices.Internal.IISVirtualRoot' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.EnterpriseServices.Internal.IServerWebConfig' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.EnterpriseServices.Internal.ISoapClientImport' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.EnterpriseServices.Internal.ISoapServerTlb' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.EnterpriseServices.Internal.ISoapServerVRoot' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.EnterpriseServices.Internal.ISoapUtility' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.EnterpriseServices.Internal.Publish' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.EnterpriseServices.Internal.ServerWebConfig' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.EnterpriseServices.Internal.SoapClientImport' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.EnterpriseServices.Internal.SoapServerTlb' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.EnterpriseServices.Internal.SoapServerVRoot' does not exist in the implementation but it does exist in the contract.
-TypesMustExist : Type 'System.EnterpriseServices.Internal.SoapUtility' does not exist in the implementation but it does exist in the contract.
-Total Issues: 104
+Total Issues: 0
```

Comment:
The surface area I based myself with is actually an empty assembly, doesn't even have type forwards or internal classes.